src/main/java/com/esri/samples/localserver/local_server_dynamic_workspace_shapefile/README.md

```diff
@@ -17,15 +17,15 @@
 <ol>
 <li>Create and run a local server.
 <ul><li><code>LocalServer.INSTANCE</code> creates a local server</li></ul>
-<ul><li><code>Server.startAsync()</code> starts the server asynchronously</li></ul>
+<ul><li><code>Server.startAsync()</code> starts the server asynchronously</li></ul></li>
```

Comment:
I think the `ul` tags are wrong. Should be
```html
<li>Create and run a local server.
<ul>
<li><code>LocalServer.INSTANCE</code> creates a local server</li>
<li><code>Server.startAsync()</code> starts the server asynchronously</li>
</ul>
```

Reply:
Fixed per your recommendation.